.github/workflows/docker.yml

```diff
@@ -18,22 +18,23 @@ permissions:
   packages: write
 
 jobs:
+  fetch_ci_data:
+    name: 📥 Fetch CI data
+    runs-on: ubuntu-latest
+    outputs:
+      toolchains-data: ${{ steps.data.outputs.toolchains-data }}
+    steps:
+      - id: data
+        run: |
+          curl -s https://raw.githubusercontent.com/oxidize-rb/rb-sys/$GITHUB_SHA/data/toolchains.json > toolchains.json
+          echo "toolchains-data=$(jq -c '.toolchains' toolchains.json)" >> $GITHUB_OUTPUT
   docker_images:
+    needs: fetch_ci_data
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        platform:
-          - ruby_target: x86_64-linux
-          - ruby_target: x86_64-linux-musl
-          - ruby_target: aarch64-linux
-          - ruby_target: aarch64-linux-musl
-          - ruby_target: arm-linux
-          - ruby_target: x86_64-darwin
-          - ruby_target: arm64-darwin
-          - ruby_target: x64-mingw32
-          - ruby_target: x64-mingw-ucrt
-          # - ruby_target: x86-linux
+        toolchain: ${{ fromJSON(needs.fetch_ci_data.outputs.toolchains-data) }}
```

Comment:
This will include `x86-linux` again; do we want to omit it?

Reply:
if it builds, i'm ok with it! there was an issue before breaking builds :/